### DISCUSSION TITLE
The export function cannot display Chinese properly

### DISCUSSION THREAD

**Discussion by gaga10086** [firefly-iii/firefly-iii#7798] [Firefly III]
![image](https://github.com/firefly-iii/firefly-iii/assets/88051070/0f2dfe6e-e14b-4b51-93fc-a03e2e9a1167)
When I export all transactions and metadata from Firefly III, a .csv file is automatically downloaded, but this file shows garbled characters and cannot display Chinese correctly. Can you tell me what I need to do? Thank you so much.

**Comment by JC5** [comment]
Did you import it as UTF8?
